src/program.ts

```diff
@@ -668,6 +668,47 @@ export class Program extends DiagnosticEmitter {
     );
   }
 
+  makeNativeMember(
+    name: string,
+    type: string
+  ): DeclarationStatement {
+    return Node.createFieldDeclaration(
+      Node.createIdentifierExpression(name, this.nativeSource.range), //name: IdentifierExpression,
+      Node.createNamedType(
+        Node.createSimpleTypeName(type, this.nativeSource.range), //name: TypeName,
+        null,//typeArguments: TypeNode[] | null,
+        false,//isNullable: bool,
+        this.nativeSource.range,//range: Range
+      ), //type: TypeNode | null,
+      null,//initializer: Expression | null,
+      null,//decorators: DecoratorNode[] | null,
+      CommonFlags.INSTANCE,//flags: CommonFlags,
+      this.nativeSource.range//range: Range
+    )
+  }
+
+  makeNativeClassDeclaration(name: string, members: DeclarationStatement[]): ClassDeclaration {
+    return Node.createClassDeclaration(
+      Node.createIdentifierExpression(name, this.nativeSource.range), //identifier: IdentifierExpression,
+      null,//typeParameters: TypeParameterNode[] | null,
+      null,//extendsType: NamedTypeNode | null, // can't be a function
+      null,//implementsTypes: NamedTypeNode[] | null, // can't be functions
+      members,//members: DeclarationStatement[],
+      null,//decorators: DecoratorNode[] | null,
+      0,//flags: CommonFlags,
+      this.nativeSource.range//range: Range
+    )
+  }
+
+  makeNativeClassPrototype(name: string, members: DeclarationStatement[]): ClassPrototype | null {
+    return this.initializeClass(
+      this.makeNativeClassDeclaration(name, members),
+      this.nativeFile,
+      [],
+      []
+    )
+  }
+
```

Comment:
These let us generate some boilerplate IR for the class which is required to compile it, pointing the source at 'native' rather than a real file like in a non-generated source file.

Reply:
No longer need these, class generation has been removed